libs/community/tests/integration_tests/vectorstores/test_vdms.py

```diff
@@ -20,6 +20,7 @@
     import vdms
```

Comment:
FYI there's a standard test suite for read / write API for vectorstores you can use it to verify that the vectorstore is working properly 

https://github.com/langchain-ai/langchain/blob/5f8858a490347dddb680fa40f877228ef0dbd325/libs/core/tests/unit_tests/vectorstores/test_in_memory.py#L15-L15

Reply:
@eyurtsev I fixed other issues. Can we proceed without updating the entire test suite this time. Or is this required?

Reply:
Not required, but it will help speed up future reviews